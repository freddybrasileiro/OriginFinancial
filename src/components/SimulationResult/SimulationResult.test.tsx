import * as React from 'react';
import { shallow } from 'enzyme';
import { SimulationResult } from './index';

describe('SimulationResult', () => {
  it('Should render the SimulationResult component', () => {
    expect(
      shallow(<SimulationResult totalAmount={25000} reachGoalBy={[4, 2024]} />)
    ).toMatchSnapshot();
  });
});
