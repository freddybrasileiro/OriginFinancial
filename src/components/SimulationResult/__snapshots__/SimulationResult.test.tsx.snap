// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimulationResult Should render the SimulationResult component 1`] = `ShallowWrapper {}`;
